.storybook/preview-head-vrt.html

```diff
@@ -18,4 +24,15 @@
 		animation: none !important;
 		transition: none !important;
 	}
+
+	/**
+	 * An alternative scenario is that the VRT scenario does not render correctly unless the animation is active but paused.
+	 * In this case, we can apply the following .googlesitekit-vrt-animation-paused class to a story wrapper element to pause animations for a given story.
+	 * Please note, this only applies to animations and not transitions which don't have an equivalent paused state.
+	 */
+	#root .googlesitekit-vrt-animation-paused *,
```

Comment:
This class has been added to allow the `Loading` VRT scenarios to display properly...